protocol/src/main/java/com/amazon/opendistroforelasticsearch/sql/protocol/response/format/CsvResponseFormatter.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.protocol.response.format;
+
+import com.amazon.opendistroforelasticsearch.sql.protocol.response.QueryResult;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Singular;
+
+@RequiredArgsConstructor
+public class CsvResponseFormatter implements ResponseFormatter<QueryResult> {
+  private static final String INLINE_SEPARATOR = ",";
+  private static final String INTERLINE_SEPARATOR = System.lineSeparator();
+  private static final Set<String> SENSITIVE_CHAR = ImmutableSet.of("=", "+", "-", "@");
+
+  private final boolean sanitize;
+
+  public CsvResponseFormatter() {
+    this.sanitize = true;
+  }
+
+  @Override
+  public String format(QueryResult response) {
```

Comment:
In case we already have CsvResult abstraction. Could we make the  code like following
```
CsvResult result = new CsvResult(response)
return result.getCsv()
```